src/runtime/pay-client.js

```diff
@@ -250,8 +254,10 @@ export class PayClientBindingPayjs {
         environment: '$payEnvironment$',
         'i': {
           'redirectKey': this.redirectVerifierHelper_.restoreKey(),
+          'googleTransactionId': googleTransactionId,
```

Comment:
Let's do one or the other to avoid confusion. Since you decided to update `PaymentAsyncClient.googleTransactionId`, let's go with that.